youtube_dl/extractor/markiza.py

```diff
@@ -1,21 +1,23 @@
 # coding: utf-8
 from __future__ import unicode_literals
 
+# update 15.01.2022 jastrab
 import re
+import json
 
 from .common import InfoExtractor
-from ..compat import compat_str
 from ..utils import (
     orderedSet,
-    parse_duration,
-    try_get,
+    url_or_none,
+    determine_ext
 )
 
 
 class MarkizaIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?videoarchiv\.markiza\.sk/(?:video/(?:[^/]+/)*|embed/)(?P<id>\d+)(?:[_/]|$)'
+    _VALID_URL = r'https:\/\/(?:www\.)?videoarchiv\.markiza\.sk\/(?:video\/(?:[^\/]+\/)*|embed\/)epizoda\/(?P<id>\d+)(?:[\_\/\-]|$)'
```

Comment:
* it's not necessary to escape `/`. That's a JS (etc) thing `var regex = /https:\/\/.../;` -- I think it's clear why escaping is needed there;
* `[\_/\-]` can be `[_/-]` (`-` in a regex `[]` must be the last character);
* there's no need to unsupport http://.


Thus:
```
    _VALID_URL = r'https?://(?:www\.)?videoarchiv\.markiza\.sk/(?:video/(?:[^/]+/)*|embed/)epizoda/(?P<id>\d+)(?:[_/-]|$)'
```